scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/Handshaker.java

```diff
@@ -299,7 +306,8 @@ protected Handshaker(boolean isClient, int initialMessageSeq, DTLSSession sessio
 		this.session.setMaxTransmissionUnit(maxTransmissionUnit);
 		this.applicationLevelInfoSupplier = config.getApplicationLevelInfoSupplier();
 		this.inboundMessageBuffer = new InboundMessageBuffer();
-
+		long timeout = config.getRetransmissionTimeout();
+		this.expiresNanosTimeout = TimeUnit.MILLISECONDS.toNanos(timeout << (config.getMaxRetransmissions() + 1));
```

Comment:
Any chance to write this without `<<` ?

Reply:
Would adding a comment be enough?

Reply:
About  the content : 
I guess you try to calculate the maximum time to wait before to consider the message as timed-out.
But I'm not sure to understand the math, e.g. for max retransmission = 3
`t + t*2 + t*2*2 + t*2*2*2 = t*(1+2+4+8) = t*15` **!=** `t*16 =  t*2^(3+1) = t <<(3+1)`
Did I miss something ? 
And by the way the DTLS specification [limit the waiting time to 60s](https://tools.ietf.org/html/rfc6347#section-4.2.4.1).
So maybe it should be more like :  `min(t,60)+min(t*2,60) + ... `

About the form : 
 If I'm correct about the content, I feel that  *2^a is nearer from the original idea than << a

Reply:
About the 60s: OK, do it in a other small PR.

With that, the calculation will be adapted to a "loop", to address the 60s limit. 
By the way, the expire timeout should be some larger than the usual sum of the retransmit times to ensure, that usually a handshake timesout prior to expire.